README.md

```diff
@@ -1,3 +1,5 @@
+For more detailed information, updates, and discussions, please visit the [Lexicon Discussions](https://github.com/lexiconhq/lexicon/discussions) section on GitHub.
```

Comment:
Move this below `### Project Roadmap` and write it like this

```md
### Discussions

For more detailed information, updates, and discussions, please visit the [Lexicon Discussions](https://github.com/lexiconhq/lexicon/discussions) section on GitHub.
```

Reply:
ok